t/00unit/lib/common/absprio.c

```diff
@@ -66,7 +66,20 @@ static void test_parser(void)
     ok(p.incremental == 1);
 }
 
+void test_urgency_to_weight(void)
+{
+    ok(h2o_absprio_urgency_to_chromium_weight(0) == 256);
+    ok(h2o_absprio_urgency_to_chromium_weight(1) == 220);
+    ok(h2o_absprio_urgency_to_chromium_weight(2) == 183);
+    ok(h2o_absprio_urgency_to_chromium_weight(3) == 147);
+    ok(h2o_absprio_urgency_to_chromium_weight(4) == 110);
+    ok(h2o_absprio_urgency_to_chromium_weight(5) == 74);
+    ok(h2o_absprio_urgency_to_chromium_weight(6) == 34);
```

Comment:
Maybe 37?

Reply:
Nice catch, totally missed this, thank you! Fixed in 9c208a2.